wordpress-mobile/WordPress-iOS
# Updates to use the serif font method from WordPress Shared

Shared PR: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/266

### To test:
1. Launch the app using iOS 13
2. Tap on the Reader tab
3. The title of each post should appear using semibold New York font
4. Launch the app using iOS 12 or iOS 11
5. Tap on the Reader tab
6. The title of each post should be in the bold Noto font

## Screenshots
| 13.3 | 12.4 | 11.4 | 
|---|---|---|
|<img src="https://user-images.githubusercontent.com/793774/85881614-c2d3f380-b792-11ea-9d69-85de69c59ffb.png"/>|<img src="https://user-images.githubusercontent.com/793774/85882090-80f77d00-b793-11ea-91a1-f68a8180580a.png" />|<img src="https://user-images.githubusercontent.com/793774/85881612-c0719980-b792-11ea-8bb7-17c9f15afd86.png" />|

### PR submission checklist:
